extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,10 +68,19 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
```

Comment:
Can you add comments on what happens with regard to laying out children when they are not visible? Is everything layedout regardless of visibility? (also create an issue and mention it here so we can fix the layout problems when the layers work is in)